### ISSUE TITLE
[request] Prevent player status colour from overriding ship type colour

### ISSUE THREAD

**Issue by JohanAR** [AndrewTaro/TTaroTeamPanel#11]
Great mod, really helps people like me who can't memorize the stats for every ship in the game!

Would it be possible to add an option to not colour ship type by status, i.e. pink and gold for self, and only use ship type colour when enabled?

**Comment by AndrewTaro** [comment]
I can't promise the addition since it's not a common request, but I'll consider it.

**Comment by JohanAR** [comment]
I created a PR since I wanted to implement this for my self either way :)

Never done any WoWs modding before so I tried to guess how it's done by looking at the surrounding code. Toggling golden colour for self appears to be working, but I didn't get into a match with anyone having tk status so that isn't tested.

**Comment by AndrewTaro** [comment]
The merge still has to wait for a while.

The reason and main problem is, if I keep accepting all the suggestions without considering the popularity, the Mod Config will easily have 10000+ options for every single mod.

**Comment by JohanAR** [comment]
Aight, no worries. Do you know how popular class colour themes are, or if most people don't use one? Perhaps those who do haven't given much thought to combining a theme with status colours, and would be equally happy if the status colours always were disabled with a theme active :)

**Comment by AndrewTaro** [comment]
As far as I'm aware, many people prefer using the color themes. (but no one has raised the feedback about the status color so far. That's why I'm concerned about the popularity)

as you assumed, probably not many people have thought about it. I suppose some of the reasons are that
- There are few "pink" players nowadays
- The other team panels behave similarly.
I'm investigating how much this feature is wanted :)

**Comment by JohanAR** [comment]
Much appreciated! I don't want to give you any extra work, so let me know if there's anything I can help with.

I think it's the Aslain v2 theme I'm using, with yellow cruisers and pink-ish subs so they were quite close to the status colours. Perhaps there's less confusion with other themes

**Comment by JohanAR** [comment]
Hmm, could've sworn I posted these comparison screenshots a few days ago, but I must've forgot to click the submit button..

Current:
![Screenshot_20240731_113102](https://github.com/user-attachments/assets/bf31cc4c-eb87-4102-b6e4-90c23f3ac2e2)

With change (if enabled):
![Screenshot_20240731_113046](https://github.com/user-attachments/assets/3420205e-de21-4dea-8d7b-b7ffa48926c7)

**Comment by JohanAR** [comment]
Sorry for the spam, but have you noticed Github acting weird? When I wrote the post above I could see the comments on my PR but not the one containing the screenshots, which is why I posted them again here. When I checked yesterday it was there, but this entire issue was completely gone. The issue/feature request didn't show up regardless of filtering for open or closed.. And today all the comments on my PR are gone but now I can see this issue again.
